sonar-php-plugin/src/main/java/org/sonar/plugins/php/PhpPlugin.java

```diff
@@ -106,6 +107,20 @@ public void define(Context context) {
           .subCategory(PHPUNIT_SUBCATEGORY)
           .build());
     }
+    if (context.getSonarQubeVersion().isGreaterThanOrEqual(SQ_VERSION_6_2)) {
```

Comment:
@inverno IMO it would be useful to add deprecation message to description of deprecated properties (e.g. in SonarJS https://github.com/SonarSource/sonar-javascript/blob/master/sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/JavaScriptPlugin.java#L82)